paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -204,3 +216,8 @@ class PaparazziPlugin : Plugin<Project> {
 }
 
 private const val DEFAULT_COMPILE_SDK_VERSION = 30
+
+private fun ResolvableDependencies.artifactViewFor(attrValue: String): ArtifactView = artifactView {
+  it.isLenient = true
```

Comment:
what does this do?  is it necessary?

Reply:
Yes it is necessary. Without this, Gradle will throw an exception if any dependencies have no variants that match the attribute we are looking for. 

Here's an portion of the stacktrace:
```
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':sample:preparePaparazziDebugResources'.
> Could not resolve all files for configuration ':sample:debugRuntimeClasspath'.
   > No variants of org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.5.31 match the consumer attributes:
       - org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.5.31 configuration runtime declares a runtime of a component:
           - Incompatible because this component declares a component, as well as attribute 'artifactType' with value 'jar' and the consumer needed a component, as well as attribute 'artifactType' with value 'android-symbol-with-package-name'
           - Other compatible attributes:
               - Doesn't say anything about com.android.build.api.attributes.BuildTypeAttr (required 'debug')
               - Doesn't say anything about org.jetbrains.kotlin.platform.type (required 'androidJvm')
```

and here's the docs for `isLenient`: 
```
Specify if the view should be resolved in a 'lenient' fashion. 
When set to true, this view will resolve as many artifacts and/or files as possible collecting any failures. 
When set to false, any failures will be propagated as exceptions when the view is resolved.
```